### PR TITLE
[chore] Update workbox to 5.1.2

### PR DESCRIPTION
**Pull request by sirugh**
## Description
Workbox logging is better in v5.

Review breaking changes [here](https://github.com/GoogleChrome/workbox/releases/tag/v5.0.0).

## Related Issue

Closes #issue.

### Verification Steps

1. Go to deployed page with console closed.
2. Open console. Should see no workbox error.
3. Refresh the page. Should see no workbox error.

## Checklist

- [ ] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Comment by sirugh** [comment]
Apparently the workbox team is working on v6. If the ETA for that initial release is close it's probably not worth doing an intermediate upgrade to v5. I'll see if I can find out when to expect v6. Maybe @zetlen knows?

File: packages/venia-concept/src/ServiceWorker/setupWorkbox.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
+import { skipWaiting, clientsClaim } from 'workbox-core';
+import { precacheAndRoute } from 'workbox-precaching';
+
 export default function() {
     /**
      * Import and Instantiate workbox object.
      */
     importScripts(
-        'https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js'
+        'https://storage.googleapis.com/workbox-cdn/releases/5.1.2/workbox-sw.js'
```
Comment (jimbo) [inline packages/venia-concept/src/ServiceWorker/setupWorkbox.js:9]: Does this version (`5.1.2`) differ from the version listed in `package.json` (`5.1.3`) for any particular reason?

Comment (sirugh) [reply packages/venia-concept/src/ServiceWorker/setupWorkbox.js:9]: Oh that was actually my bad - I need to update this. Thanks for reminding me :)

File: packages/venia-concept/src/ServiceWorker/Utilities/__tests__/imageCacheHandler.spec.js [hunk 2]
```diff
@@ -11,6 +13,24 @@ import {
...
+
+jest.mock('workbox-expiration', () => {
+    return {
+        ExpirationPlugin: jest.fn().mockImplementation()
+    };
+});
+
+jest.mock('workbox-strategies', () => {
+    return {
+        CacheFirst: jest.fn().mockImplementation()
```
Comment (supernova-at) [inline packages/venia-concept/src/ServiceWorker/Utilities/__tests__/imageCacheHandler.spec.js:30]: Is `jest.fn().mockImplementation()` the same as `jest.fn()`?

Comment (sirugh) [reply packages/venia-concept/src/ServiceWorker/Utilities/__tests__/imageCacheHandler.spec.js:30]: Oh, yes I think so. https://jestjs.io/docs/en/mock-function-api#mockfnmockimplementationfn

> Note: jest.fn(implementation) is a shorthand for jest.fn().mockImplementation(implementation).

**Pull request merged**